src/main/java/duke/exception/DukeParseException.java

```diff
@@ -0,0 +1,7 @@
+package duke.exception;
```

Comment:
I like how you created different classes to represent different forms of exceptions! Good job for your hard work! 😄